packages/next/build/babel/plugins/next-ssg-transform.ts

```diff
@@ -3,18 +3,22 @@ import * as BabelTypes from '@babel/types'
 
 const pageComponentVar = '__NEXT_COMP'
 const prerenderId = '__N_SSG'
+const serverPropsId = '__N_SPROPS'
```

Comment:
nit: how about this?
```suggestion
const serverPropsId = '__N_SSP'
```

Reply:
That seems good 👍